### PR TITLE
Bash

### PR DESCRIPTION
**Pull request by joellebegin**
making some documents to introduce people to bash scripts

### CODE REVIEW

File: bash_scrips/intro_to_bash.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# A brief introduction to bash scripting
+
```
Comment (acliu) [inline bash_scrips/intro_to_bash.md]: I think we should add the author's name and the date below each of our little tutorials.

File: bash_scrips/distributions.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    poisson = np.random.poisson(size = 2000)
+    gaussian = np.random.normal(size = 2000)
+    pareto = np.random.pareto(3, size = 2000)
+
+    np.savetxt('poisson_distribution_2000_datapoints.txt', poisson, delimiter=',')
+    np.savetxt('gaussian_distribution_2000_datapoints.txt', gaussian, delimiter=',')
+    np.savetxt('pareto_distribution_2000_datapoints.txt', pareto, delimiter=',')
+
+if __name__ == "__main__":
+    main()
```
Comment (acliu) [inline bash_scrips/distributions.py]: Convention is to have a new line at the end of files.

File: bash_scrips/histograms.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    file_loc = input()
+    data = np.loadtxt(file_loc, delimiter = ',')
+
+    fig, ax = plt.subplots()
+    ax.hist(data, density = True)
+
+    plt.savefig(file_loc + '.png')
+
+if __name__ == "__main__":
+    main()
```
Comment (acliu) [inline bash_scrips/histograms.py]: Convention is to have a new line at the end of files.

**Review by acliu — CHANGES_REQUESTED** [review]
Another thing: I assume you meant to call the folder `bash_scripts` rather than `bash_scrips`?

**Comment by joellebegin** [comment]
Hi Adrian I satisfied all the change requests (this is long overdue, I know) but I don't know how to tell github that it's fixed :( I don't know if you have to review again before merging.

**Review by acliu — APPROVED** [review]

**Pull request merged**